src/basic/macro.h

```diff
@@ -139,11 +139,17 @@ static inline unsigned long ALIGN_POWER2(unsigned long u) {
         return 1UL << (sizeof(u) * 8 - __builtin_clzl(u - 1UL));
 }
 
+#ifndef __COVERITY__
+#  define VOID_0 ((void)0)
+#else
+#  define VOID_0 ((void*)0)
+#endif
```

Comment:
tbh i sympathize with coverity there. I don't really know what `(void) 0` is supposed to mean, and I figure the fact that this trips up compilers is actually the intention here...